### PR TITLE
chore: add 0.10.2 CHANGELOG

### PR DESCRIPTION
**Pull request by stevenpyzhang**
### Description 
created with changelog tool between current 6.0.0 and 0.10.1-ksqldb branch

### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

**Review by JimGalasyn — APPROVED** [review]
LGTM! I'll cherry-pick to 0.10.2-ksqldb, 0.11.x-ksqldb, 0.11.0-ksqldb, 0.12.x-ksqldb, 0.12.0-ksqldb, and 0.13.x-ksqldb.

File: CHANGELOG.md [hunk 1]
```diff
@@ -74,6 +74,29 @@
...
+* allow expressions in flat map ([#6165](https://github.com/confluentinc/ksql/pull/6165)) ([b6ad9bc](https://github.com/confluentinc/ksql/commit/b6ad9bccf1fbac1ade2c586fa6bd40b33b19d35a))
+* allow joins in windowed aggregations ([9452ab6](https://github.com/confluentinc/ksql/commit/9452ab6d072882264c60042b655af52c5dc07cbf)), closes [#5898](https://github.com/confluentinc/ksql/issues/5898) [#5931](https://github.com/confluentinc/ksql/issues/5931)
+* always use the changelog subject in table state stores ([#5823](https://github.com/confluentinc/ksql/pull/5823)) ([#5837](https://github.com/confluentinc/ksql/issues/5837)) ([c87aa69](https://github.com/confluentinc/ksql/commit/c87aa6940aa480dbecf1393ff16a70d53a9d51d5))
+* avoid losing cause of processing errors ([#5937](https://github.com/confluentinc/ksql/pull/5937)) ([014c049](https://github.com/confluentinc/ksql/commit/014c049cfd6b964ae63ac39185ac44b7370c5fbb))
+* close query on invalid use of HTTP/2 with /query endpoint (6.0.x) ([#5885](https://github.com/confluentinc/ksql/pull/5885)) ([0b75411](https://github.com/confluentinc/ksql/commit/0b75411628df58ccb68ca04501e99828e63b22f3))
```
Comment (vcrfxia) [inline CHANGELOG.md:87]: nit: remove `(6.0.x)`?

**Review by vcrfxia — APPROVED** [review]
Thanks @stevenpyzhang !

File: CHANGELOG.md (hunk 1 above)
Comment (stevenpyzhang) [reply CHANGELOG.md]: removed

**Pull request merged**